node/coroutines/src/client_task.rs

```diff
@@ -194,15 +220,17 @@ impl Stream for ClientTask {
 impl ClientTask {
     pub fn new(
         client: Arc<Client>,
-        incoming_block_rx: Receiver<(SignedBeaconBlock, SignedShardBlock)>,
-        out_block_tx: Sender<(SignedBeaconBlock, SignedShardBlock)>,
+        incoming_block_rx: Receiver<(PeerId, (SignedBeaconBlock, SignedShardBlock))>,
+        out_block_tx: Sender<(Vec<PeerId>, (SignedBeaconBlock, SignedShardBlock))>,
         consensus_rx: Receiver<ConsensusBlockProposal>,
         control_tx: Sender<Control>,
         retrieve_payload_rx: Receiver<(AuthorityId, CryptoHash)>,
         payload_request_tx: Sender<PayloadRequest>,
         payload_response_rx: Receiver<PayloadResponse>,
         inc_payload_gossip_rx: Receiver<PayloadGossip>,
         out_payload_gossip_tx: Sender<PayloadGossip>,
+        inc_peer_rx: Receiver<(PeerId, ChainState)>,
```

Comment:
I would rename `inc_peer_*` and `out_peer_*` to something more descriptive that reflects what information is sent. E.g. `out_peer_*` -> `out_block_fetch_*` and `inc_peer_*` -> `inc_chain_state_*`.

Reply:
Done.